frontend/src/api/authRequest.tsx

```diff
@@ -17,9 +22,13 @@ export class AuthApiClient extends HttpClient implements IAuthApiClient {
 
   async login(payload: IUserDetails): Promise<IUserAuthApiLoginResponse>{
     return await this.post('token', payload, changedHttpRequestOptions) as IUserAuthApiLoginResponse
-}
+  }
 
-    async postRefresh(payload: ITokenSendType): Promise<ITokenRefreshType> {
-        return await this.post('token/refresh', payload, changedHttpRequestOptions) as ITokenRefreshType
-    }
+  async postRefresh(payload: ITokenSendType): Promise<ITokenRefreshType> {
+    return await this.post('token/refresh', payload, changedHttpRequestOptions) as ITokenRefreshType
+  }
+
+  async recover(payload: IUserEmail): Promise<IMessageResponse> {
+    return await this.post('recover', payload, changedHttpRequestOptions) as IMessageResponse
```

Comment:
Maybe something not merged well on my computer, but the recover of password is not working  with error `500` -
`TypeError at /api/v1/recover/
post() takes 1 positional argument but 2 were given`

Reply:
I might have broken something in my backend code while trying to fix linter errors... I will fix this right now.

Reply:
Fixed.